src/device-registry/controllers/kafka/consumers/raw-device-measurements-consumer.js

```diff
@@ -0,0 +1,52 @@
+const kafka = require('kafka-node');
+const Consumer = kafka.Consumer;
```

Comment:
Why are we adding so many subfolders inside this controllers directory? Having a length path names is not very encouraged. Anyway we can have most files just under the controllers directory without the subfolders like: `kafka/consumers`